modules/virt-about-hot-plugging-virtual-disks.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-hot-plugging-virtual-disks_{context}"]
+= About hot-plugging virtual disks
```

Comment:
```suggestion
= About hot plugging virtual disks
```
See the RHSCG word list here: https://redhat-documentation.github.io/supplementary-style-guide/#_h

Reply:
Oh wow, I had no idea - I had checked the ISG and assumed we were following that guidance. Thank you for letting me know!